nncf/common/factory.py

```diff
@@ -45,16 +39,20 @@ def create_exporter(model: ModelType,
 def create_bn_adaptation_algorithm_impl(data_loader: NNCFDataLoader,
                                         num_bn_adaptation_steps: int,
                                         num_bn_forget_steps: int,
-                                        device: Optional[str] = None) -> BatchnormAdaptationAlgorithmImpl:
+                                        device: Optional[str] = None):
```

Comment:
Let's move create_bn_adaptation_algorithm_impl to nncf.common.batchnorm_adaptation.

Reply:
Done.